#! /bin/sh /usr/share/dpatch/dpatch-run
## 1_config.dpatch by  <ballombe@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: /usr/doc moved to /usr/share/doc

@DPATCH@
diff -urNad boa-0.94.14rc21~/examples/boa.conf boa-0.94.14rc21/examples/boa.conf
--- boa-0.94.14rc21~/examples/boa.conf	2003-02-02 06:02:22.000000000 +0100
+++ boa-0.94.14rc21/examples/boa.conf	2006-11-27 14:36:48.000000000 +0100
@@ -232,7 +232,7 @@
 # Aliases: Aliases one path to another.
 # Example: Alias /path1/bar /path2/foo
 
-Alias /doc /usr/doc
+Alias /doc /usr/share/doc
 
 # ScriptAlias: Maps a virtual path to a directory for serving scripts
 # Example: ScriptAlias /htbin/ /www/htbin/
